main.go

```diff
@@ -211,8 +211,15 @@ func main() {
 		os.Exit(1)
 	}
 
-	dc := depClient(logger, *flDEPCK, *flDEPCS, *flDEPAT, *flDEPAS, *flDEPServerURL, *flDEPsim)
-	mgmtSvc := management.NewService(deviceDB, workflowDB, dc, pushSvc, appsDB, certsDB)
+	dc, err := depClient(logger, *flDEPCK, *flDEPCS, *flDEPAT, *flDEPAS, *flDEPServerURL, *flDEPsim)
+	var mgmtSvc management.Service
+	if err != nil {
```

Comment:
You're suppressing the error here, there's potential for the client to fail for other reasons besides invalid credentials.
If we want `dep` to be opt-in, I'd rather start without depClient by default. Have the user pass a flag with DEP credentials, otherwise start the server without DEP.